### PR TITLE
Fix Detected a division by zero in Animated.divide node

### PR DESCRIPTION
**Pull request by chaitanya0bhagvan**
On first render width is 0 which is causing Detected a division by zero in Animated.divide node.

### CODE REVIEW

**Comment by developer239** [comment]
+1 this is a major issue

File: index.js [hunk 1]
```diff
@@ -329,7 +329,7 @@ const ScrollableTabView = createReactClass({
   _handleLayout(e) {
     const { width, } = e.nativeEvent.layout;
 
-    if (Math.round(width) !== Math.round(this.state.containerWidth)) {
+    if (width > 0 && Math.round(width) !== Math.round(this.state.containerWidth)) {
```
Comment (syzer) [inline index.js]: Did you consider returning early ?
ex:
```js
 if (!width || Math.round(width) === Math.round(this.state.containerWidth) { return }
```

**Comment by chaitanya0bhagvan** [comment]
@syzer returning early now.

**Comment by developer239** [comment]
@chaitanya0bhagvan Why did you delete the branch? It still does not work.

**Comment by chaitanya0bhagvan** [comment]
@adamski This is already merged but a new version is not pushed to npm.

**Comment by adamski** [comment]
Ok great, I'll use the git url instead, thanks

**Pull request merged**